web/frontend/vite.lib.config.ts

```diff
@@ -1,30 +1,27 @@
 // This vite configuration builds RC in library mode for NPM distribution.
-import { defineConfig } from 'vite';
-import pkg from './package.json';
-import path from 'node:path';
-import url from 'node:url';
-import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js';
-import { plugins } from './vite.config'
+import { defineConfig } from "vite";
```

Comment:
Unintentional formatting change from `'` to `"`?

Reply:
Hm prettier is set up as my formatter, wonder why it did that.